tasks/main.yml

```diff
@@ -247,13 +247,31 @@
   tags:
   - odcs
 
+- name: Install the ODCS Cleanup systemd service
+  copy:
+    src: "odcs-celery-cleanup.service"
+    dest: "/etc/systemd/system/odcs-celery-cleanup.service"
+    owner: root
+    group: root
+    mode: 0644
+  tags:
```

Comment:
I'll just add a register: and then a `systemd` daemon_reload: True to ensure systemd knows about new .service/unit before starting it